standard/classes.md

```diff
@@ -3480,8 +3480,8 @@ A set accessor corresponds to a method with a single value parameter of the prop
 Based on the presence or absence of the get and set accessors, a property is classified as follows:
 
 - A property that includes both a get accessor and a set accessor is said to be a ***read-write property***.
-- A property that has only a get accessor is said to be a ***read-only property***. It is a compile-time error for a read-only property to be the target of an assignment.
-- A property that has only a set accessor is said to be a ***write-only property***. Except as the target of an assignment, it is a compile-time error to reference a write-only property in an expression.
+- A property that has only a get accessor is said to be a ***read-only property***. It is a compile-time error for a read-only property to be the target of an assignment unless the property is ref-valued and returns a writeable reference.
```

Comment:
The added text equates to stating that it is an error to write to a read-only property unless it isn't a read-only property…

That doesn’t make sense.

A property is read-only if:

- it is a non-ref property and only has a get accessor; or
- it is a ref property and marked readonly

C# started with one kind of property and one kind of indexer, now it has two – ref properties and ref indexers which behave more like array indexing – which though they don’t have an indexer *per se* have an indexing operation whose return is classified as a variable aka. a variable reference just like ref properties and ref indexers.